src/pages/Login/Login.jsx

```diff
@@ -13,11 +19,8 @@ export default function Login() {
         <S.SignupLink to="/signup">회원가입</S.SignupLink>
         <S.Text>SNS계정으로 간편 로그인/회원가입</S.Text>
         <section>
-          <Link to="">
-            <S.LoginImg
-              src="https://cdn-icons-png.flaticon.com/512/3669/3669973.png"
-              alt="kakaoLogin"
-            />
+          <Link onClick={handleLogin}>
+            <S.LoginImg src="./images/soomin/withkakao.png" alt="kakaoLogin" />
```

Comment:
```suggestion
            <S.LoginImg src="/images/soomin/withkakao.png" alt="kakaoLogin" />
```
public 하위에 images에 접근하는 정적 이미지는 절대경로로 수정 해주세요